### PR TITLE
Mobile version update

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
This is the feature update to complete the mobile version of my Portfolio
1. Added two project grids to my works section
2. Added "About me" section with a get me button
3. Added list of credentials and languages in the footer section before the contact forms

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -54,5 +54,113 @@ <h2>I am a Software Developer</h2>
...
+          />
+          <img
+            src="/images/Icons/languages/darkicons/bootstrap.png"
+            alt="bootlang"
+          />
+          <img
+            src="/images/Icons/languages/darkicons/ruby.png"
+            alt="rubylang"
+          />
+        </div>
```
Comment (Danieldotcomcoder) [inline portfolio.html]: Your works section doesn't match the Figma design, you should try to stick to the design as possible, please fix this : 👍😃

image

Figma Design : 
image

File: portfolio.html [hunk 2]
```diff
@@ -54,5 +54,113 @@ <h2>I am a Software Developer</h2>
...
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div id="lang">
+        <img src="/images/Icons/languages/css.png" alt="csslang" />
+        <img src="/images/Icons/languages/html.png" alt="htmllang" />
+        <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
+        <img src="/images/Icons/languages/ruby.png" alt="rubylang" />
+      </div>
```
Comment (Danieldotcomcoder) [inline portfolio.html]: Kindly wrap this category section in a `ul`  tag and put each element as `li` tag as stated in the project requirements, so your code will be more professional

File: portfolio.html [hunk 3]
```diff
@@ -54,5 +54,113 @@ <h2>I am a Software Developer</h2>
...
+        <h2>Ruby on Rails</h2>
+        <h2>RSPec</h2>
+        <h2>CapyBara</h2>
+        <h2>Selenium</h2>
+        <h1><span>&#9679;</span> Skills</h1>
+        <h2>Codekit</h2>
+        <h2>Github</h2>
+        <h2>Codepen</h2>
+        <h2>Gitlab</h2>
+        <h2>Terminal</h2>
```
Comment (Danieldotcomcoder) [inline portfolio.html]: The categories inside the about myself section should be inside a `ul` tag, and each one of the elements ( Languages, Frameworks, and Skills ) should be in `li` tag as stated in the requirements, Kindly fix this 👍😃

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @ramrod433 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linter are setup correctly ✔
- No Linter Errors ✔
- Professional PR ✔

## Required Changes ♻️
_Check the comments under the review._

[OPTIONAL]: It would be better if you remove this from your Readme File so it will look more professional : ✔👍
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: portfolio.html [hunk 4]
```diff
@@ -54,5 +54,127 @@ <h2>I am a Software Developer</h2>
...
+          </ul>
+        </div>
+      </div>
+      <div class="longbutton">
+        <a href="#longbotton"><img src="/images/Action.png" alt="button" /> </a>
+      </div>
+    </section>
+    <section class="about-section">
+      <div class="about">
+        <h1>About me</h1>
```
Comment (Donard97) [inline portfolio.html:135]: - [ ] Well done of implementing the required changes from the other reviewer but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Keep rocking 💪 

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

File: portfolio.html [hunk 5]
```diff
@@ -54,5 +54,127 @@ <h2>I am a Software Developer</h2>
...
+          <li><img src="/images/Icons/languages/html.png" alt="htmllang" /></li>
+          <li>
+            <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
+          </li>
+          <li><img src="/images/Icons/languages/ruby.png" alt="rubylang" /></li>
+        </ul>
+      </div>
+      <div class="project-button">
+        <a href="projectlink"
+          ><img
```
Comment (Donard97) [inline portfolio.html:92]: - [OPTIONAL] I suggest you before merging to add button interactions following the Figma design:

image

**Review by Donard97 — CHANGES_REQUESTED** [review]
Hi @ramrod433 , 👋  
    
_Great work on making the changes requested by a previous reviewer 👏 💯_ 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 💪 

giphy (5)


## Highlights 🏁 

- There is no linter error ✔️ 
- Gitflow is used correctly ✔️ 
- PR is descriptive ✔️ 
- The Readme file is professional ✔️ 
- Code is clean ✔️ 
    
## Required Changes ♻️

- [ ] Kindly add the other files for linter following linters configuration [repository](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [ ] Please check the comments under the review.
    
## Optional suggestions
    
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
    
**Cheers and Happy coding!👏👏👏**
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
------
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
 ```

**Review by Davidosky007 — COMMENTED** [review]
Hello @ramrod433  👋,

_This is an additional review as a result of the second opinion request._

- Kindly note that setting up linters locally isn't a requirement it is used only on your local system to run linter check and fix errors if there are any, but for Github actions, linter checks you only need the  .github/workflows folder and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) to that folder. so as long as the linter checks are passing on Github the project is good to go.
- About sticking to the Figma design as much as possible these include the (  font, colours, images, tests, margins) [refer to the project requirements ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20tests%2C%20margins)%20using%20the%20templates%20in%20Figma.). It is a requirement and part of this milestone learning objectives ( to be able to to parse a Figma design to create a UI)
- ## Note the content of the project doesn't have to be exactly the same as in the figma design just the ( font, colors, images, tests, margins)

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Comment by Ursaaaaaa** [comment]
Thank you sir

**Comment by smunozmo** [comment]
Hi @ramrod433 , 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 



💡  Just a friendly tip: If you name your main file `index.html` it will be easier to deploy your project in the future.

Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**